scripts/Inelastic/CrystalField/energies.py

```diff
@@ -0,0 +1,43 @@
+from mantid.simpleapi import *
```

Comment:
Odd. I just tried by building the branch and running this in IPython:
```python
from mantid.simpleapi import CrystalFieldEnergies
CrystalFieldEnergies(1, B20=0.3365, B22=7.4851, B40=0.4062, B42=-3.8296, B44=-2.3210)
```
and it worked.